examples/gnrc_border_router/README.md

```diff
@@ -8,6 +8,35 @@ and routes on the BR.
 
 The script `start_network.sh` enables a *ready-to-use* BR in only one command.
 
+## Uplink
+
+The border router will route packets between a 6Lo network (PAN) and a 'normal'
+IPv6 network (i.e. the Internet).
+
+This requires the border router to have two interfaces, a downstream, 6lo-interface
+and an IPv6 uplink.
+
+This setup comes with support for three uplink types pre-configured.
```

Comment:
The three uplink types should be listed here, the default specified and `native` introduced as a special case
```suggestion
This example comes with support for three uplink types pre-configured:
- [`ethos`](https://doc.riot-os.org/group__drivers__ethos.html) (default)
- [`slip`](https://tools.ietf.org/html/rfc1055)
- `wifi`

For `native` the host-facing [`netdev_tap`](https://doc.riot-os.org/netdev__tap_8h.html) device
is configured, providing connectivity via a TAP interface to the RIOT instance.
 
```